docs/UsersGuide/best/hub-pattern.md

```diff
@@ -17,4 +17,134 @@ ARINC 653 Channels, High-speed hardware buses between nodes, and UARTs between n
 ## Generic Hub
 
 There is now a standard implementation of the hub pattern. The [GenericHub](../api/c++/html/svc_generic_hub_component.html) is an
-implementation of the hub pattern that passes through F´ ports and `Fw::Buffer`s.
\ No newline at end of file
+implementation of the hub pattern that passes through F´ ports and `Fw::Buffer`s.
+
+## Detailed Example Generic Hub Setup
+
+A possible usage of the Hub pattern includes running F´ on multiple computers acting as one F´ system. This example could 
+serve as a base for a multi-computer system running F´.
+
+![Detailed Hub](../media/detailed_hub_example.png)
+
+In this example, a basic implementation of the [Communication Adapter Interface](../../Design/
+communication-adapter-interface.md) 
+facilitates the transfer of serialized data between the two nodes across two different deployments. This interface 
+may be reinforced by including a `Fw::ComStub` and `Fw::ComQueue`, which are excluded in this implementation for 
+simplicity. The Hub uses the `Data Out` port to export serialized data to the remote hub, which receives them on 
+the `Data In` port. This connection works reflexively, allowing the two hubs to communicate and transmit data 
+between each other.
+
+A [Command Splitter](../api/c++/html/svc_cmd_splitter.html) splits incoming commands from the Ground Interface to either a local or
+remote [Command Dispatcher](../api/c++/html/svc_cmd_dispatcher.html). This is inserted between the Ground Interface and the local Command 
+Dispatcher. The command offset of the command splitter should match the commands on the external node. An offset of 0x10000 will route any
```

Comment:
I might say `command opcode` instead of command - that way it explicitly refers to the numerical opcode